lenses/tests/test_nslcd.aug

```diff
@@ -0,0 +1,408 @@
+  }
+  {  }
+  { "#comment" = "This option allows for custom attributes to be looked up instead of the default RFC 2307 attributes." }
+  { "map"
+    { "passwd"
+      { "homeDirectory" = "\"${homeDirectory:-/home/$uid}\"" }
+    }
+  }
+  { "map"
+    { "passwd"
+      { "loginShell" = "\"${loginShell:-/bin/bash}\"" }
+    }
+  }
+  { "map"
+    { "shadow"
+      { "userPassword" = "myPassword" }
+    }
+  }
+  {  }
+  { "#comment" = "Timing/reconnect options" }
+  {  }
+  { "#comment" = "Specifies the time limit (in seconds) to use when connecting to the directory server." }
+  { "bind_timelimit" = "30" }
+  {  }
+  { "#comment" = "Specifies the time limit (in seconds) to wait for a response from the LDAP server." }
+  { "timelimit" = "5" }
+  {  }
+  { "#comment" = "Specifies the period if inactivity (in seconds) after which the connection to the LDAP server will be closed." }
+  { "idle_timelimit" = "10" }
+  {  }
+  { "#comment" = "Specifies the number of seconds to sleep when connecting to all LDAP servers fails." }
+  { "reconnect_sleeptime" = "10" }
+  {  }
+  { "#comment" = "Specifies the time after which the LDAP server is considered to be permanently unavailable." }
+  { "reconnect_retrytime" = "10" }
+  {  }
+  { "#comment" = "SSL/TLS options" }
+  {  }
+  { "#comment" = "Specifies whether to use SSL/TLS or not (the default is not to)." }
+  { "ssl" = "start_tls" }
+  { "#comment" = "Specifies what checks to perform on a server-supplied certificate." }
+  { "tls_reqcert" = "never" }
+  { "#comment" = "Specifies the directory containing X.509 certificates for peer authentication." }
+  { "tls_cacertdir" = "/etc/ssl/ca" }
+  { "#comment" = "Specifies the path to the X.509 certificate for peer authentication." }
+  { "tls_cacertfile" = "/etc/ssl/certs/ca-certificates.crt" }
+  { "#comment" = "Specifies the path to an entropy source." }
+  { "tls_randfile" = "/dev/random" }
+  { "#comment" = "Specifies the ciphers to use for TLS." }
+  { "tls_ciphers" = "TLSv1" }
+  { "#comment" = "Specifies the path to the file containing the local certificate for client TLS authentication." }
+  { "tls_cert" = "/etc/ssl/certs/cert.pem" }
+  { "#comment" = "Specifies the path to the file containing the private key for client TLS authentication." }
+  { "tls_key" = "/etc/ssl/private/cert.pem" }
+  {  }
+  { "#comment" = "Other options" }
+  { "pagesize" = "100" }
+  { "nss_initgroups_ignoreusers" = "user1,user2,user3" }
+  { "nss_min_uid" = "1000" }
+  { "nss_nested_groups" = "yes" }
+  { "nss_getgrent_skipmembers" = "yes" }
+  { "nss_disable_enumeration" = "yes" }
+  { "validnames" = "/^[a-z0-9._@$()]([a-z0-9._@$() \~-]*[a-z0-9._@$()~-])?$/i" }
+  { "ignorecase" = "yes" }
+  { "pam_authc_ppolicy" = "yes" }
+  { "pam_authz_search" = "(&(objectClass=posixAccount)(uid=$username)(|(authorizedService=$service)(!(authorizedService=*))))" }
+  { "pam_password_prohibit_message" = "\"MESSAGE LONG AND WITH SPACES\"" }
```

Comment:
I did remove the quotes for this field, as they should be always present (in the configuration file)